### ISSUE TITLE
deployment from vs code to azure webapp will not work

### ISSUE THREAD

**Issue by nick-ID** [microsoft/vscode-azureappservice#2391]
Does this occur consistently? 
Repro steps:

1.
2.

Action: appService.Deploy
Error type: TypeError
Error Message: Cannot read properties of undefined (reading 'createClient')

Version: 0.24.7
OS: win32
OS Release: 10.0.19045
Product: Visual Studio Code
Product Version: 1.75.1
Language: en

<details>
<summary>Call Stack</summary>

```
extension.bundle.js:2:3774915extension.bundle.js:2:3774915
fulfilled extension.bundle.js:2:3773586
```

</details>

**Comment by nturinski** [comment]
Hi @nick-ID,

Could you please provide detailed repro steps? And does this issue happen persistently? If you try running the command again or refreshing the tree view, does the error go away?

**Comment by alexweininger** [comment]
I believe this issue happens when the following conditions are met:
* `appService.defaultWebAppToDeploy` setting is set in the settings.json file
* Azure App Service extension is not yet activated

If you continue to face this issue, removing the `appService.defaultWebAppToDeploy` setting, or ensuring the Azure App Service extension is activated before running the deploy command should fix it.

**Comment by alexweininger** [comment]
This issue has been fixed by the [0.7.0 release of the Azure Resources extension](https://github.com/microsoft/vscode-azureresourcegroups/releases/tag/v0.7.0). Please update the Azure Resources extension to version 0.7.0.
